autogpt_platform/backend/backend/blocks/stripe_link/profile.py

```
@@ -0,0 +1,172 @@
+        *,
+        credentials: StripeLinkCredentials,
+        **kwargs: Any,
+    ) -> BlockOutput:
+        try:
+            info = await self._link_api_request(credentials, "GET", "/userinfo")
+            yield "name", info.get("name", "")
+            yield "first_name", info.get("first_name", "")
+            yield "last_name", info.get("last_name", "")
+            yield "email", info.get("email", "")
+            yield "phone", info.get("phone", "")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)